### ISSUE TITLE
Video in portfolio

### ISSUE THREAD

**Issue by gido** [antistatique/antistatique.net#280]
I played a bit with paragraph to insert a video into one of our portfolio:

http://antistatique.net/fr/nous/travaillons/avec/montreux-jazz-festival

![screen shot 2015-08-04 at 10 49 46](https://cloud.githubusercontent.com/assets/101859/9056922/935493fe-3a96-11e5-9e38-1a79f3ef9876.png)

I think it would be nice to have a wider player (currently is a `Simple body` paragraph with custom HTML inside).
And what about the new project structure proposed in #244 ? @tonifisler @zufrieden @guillaumeberry

**Comment by gido** [comment]
@tonifisler we neeeed it also for https://antistatique.net/fr/nous/travaillons/avec/cardis2017

**Comment by gido** [comment]
In the meantime @nsz @guillaumeberry : 

```html
<div class="embed-responsive embed-responsive-16by9">
  <iframe src="https://player.vimeo.com/video/201993601?color=e30074&title=0&byline=0&portrait=0" width="100%" height="100%" frameborder="0" class="embed-responsive-item" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
</div>
```
([Bootstrap documentation](http://getbootstrap.com/components/#responsive-embed))

**Comment by tonifisler** [comment]
The Bootstrap `embed-responsive` code is simpler. HTML doesn't scare us. 💪

**Comment by gido** [comment]
@tonifisler Yes, but what if we want to display larger video (not limited to the actual body col size) ?

**Comment by tonifisler** [comment]
I apparently validated that in 2015. Guess I can't argue anymore! :D
